contributors/devel/release.md

```diff
@@ -300,20 +302,21 @@ of the issue.
   - even less urgent / critical than `priority/important-soon`
   - moved out of milestone more aggressively than `priority/important-soon`
 
-### Issue Kind Label
+### Issue/PR Kind Label
```

Comment:
super nit: space between issue and PR

Reply:
I didn't think so? https://www.grammarly.com/blog/slash/